cmd/bigsky/main.go

```diff
@@ -35,7 +35,7 @@ var log = logging.Logger("bigsky")
 
 func init() {
 	//logging.SetAllLoggers(logging.LevelDebug)
-	logging.SetAllLoggers(logging.LevelWarn)
+	logging.SetAllLoggers(logging.LevelInfo)
```

Comment:
I made a note for myself to look in to this. Want to ensure that we don't clobber `GOLOG_LOG_LEVEL` env var, but also have basically all commands default to `info` verbosity.